src/System.Private.Xml/tests/XmlTextReader/System.Xml.XmlTextReader.Tests/XmlTextReaderAttributeTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Xml.Tests
+{
+    public class XmlTextReaderAttributeTests
+    {
+        [Fact]
+        public void XmlTextReaderGetAttributeWithIndexTest()
+        {
+            XmlTextReader textReader = XmlTextReaderTestHelper.CreateReader("<element1 attr='val'> abc </element1>");            
+            Assert.True(textReader.Read());
+            Assert.Equal("val", textReader.GetAttribute(0));
+        }
+
+        [Fact]
+        public void XmlTextReaderMoveToAttributeWithNameTest()
```

Comment:
please add another more test cases where: attribute is not there, there is more than one attribute with the same name (check if moved to the correct one)